support/mock/mockrt.go

```diff
@@ -64,7 +64,8 @@ type Runtime struct {
 	expectVerifyConsensusFault *expectVerifyConsensusFault
 	expectDeleteActor          *address.Address
 
-	logs []string
+	logs       []string
+	gasCharged int64
```

Comment:
Add a note that this is only the gas explicitly charged through the runtime method, and that real VMs will charge most gas implicitly.